packages/host/app/components/ai-assistant/message/index.gts

```diff
@@ -303,6 +304,21 @@ export default class AiAssistantMessage extends Component<Signature> {
         overflow: auto;
       }
 
+      .is-streaming
+        .content
+        > :not(ol):not(ul):not(pre):not(div):last-child:after {
+        content: '';
+        width: 8px;
+        height: 8px;
+        background: currentColor;
+        border-radius: 50%;
+        display: inline-block;
+        font-family: system-ui, sans-serif;
+        line-height: normal;
+        vertical-align: baseline;
+        margin-left: 5px;
+      }
+
```

Comment:
```
.is-streaming .content[data-scopedcss-b61d7f149b-f81bdaca25] > :not(div):last-child:after {
    content: '…';
}
```

It’s not the same look but does suggest continuation?

<img width="367" alt="Boxel 2025-02-04 15-42-40" src="https://github.com/user-attachments/assets/e2a9b588-412f-4b8e-b977-a32b797f277e" />